CLAUDE.md

```diff
@@ -7,21 +7,70 @@ This file provides guidance to Claude Code (claude.ai/code) when working with co
 - Frontend Build: `cd frontend && npm run build`
 - Frontend Lint: `cd frontend && npm run lint`
 - Backend Run: `cd backend && uv pip install -r requirements.txt && uvicorn app.main:app --reload --log-config=logging_config.yaml`
-- Backend Format: `cd backend && black app`
+- Backend Format: `cd backend && black app && isort app`
```

Comment:
Not super related for this, but I'd also go for ruff here instead of black/isort.